pkg/plugin/aggregation/update.go

```diff
@@ -13,7 +13,10 @@ import (
 	"github.com/heptio/sonobuoy/pkg/plugin"
 )
 
-const annotationName = "sonobuoy.hept.io/status"
+const (
+	StatusAnnotationName = "sonobuoy.hept.io/status"
+	StatusPodName        = "sonobuoy"
+)
 
 type key struct {
```

Comment:
Please document this composite key. Maybe something about how a plugin is uniquely defined by the node it runs on and its name (which isn't actually true as we would also need the sonobuoy id to uniquely define a plugin).